Sources/SwiftProtobuf/Google_Protobuf_Any+Registry.swift

```diff
@@ -45,7 +45,7 @@ internal func typeName(fromURL s: String) -> String {
     }
   }
 
-  return s[typeStart..<s.endIndex]
+  return String(s[typeStart..<s.endIndex])
```

Comment:
You mention it is a no op for Swift 3, is there any real cost to it for Swift 4 we need to worry about?